lib/taurus/qt/qtgui/taurusgui/utils.py

```diff
@@ -397,11 +413,19 @@ def fromPanel(panel):
         else:
             # ignore other "model" attributes (they are not from Taurus)
             model = None
+        icon = panel.icon
+        model_in_config = panel.model_in_config
+        modifiable_by_user = panel.modifiablebyuser
+        formatter = panel.formatter
```

Comment:
I think that the `formatter` that makes sense to store is `panel.widget.formatter`, not `panel.formatter`

Also note that the members of the panel description should be serializable in order to be able to store them as xml. In the case of the formatter, it could be a string, but it could also be a *function*. In the latter case, some conversion would be needed to serialize it as a string.

Reply:
Actually name `formatter` is not the best one for what I wanted to achieve. More descriptive would be `widget_formatter`. I've accidentally use wrong attribute of `PanelDescription` but now name is changed so name conflict is resolved.